odin/fields/__init__.py

```diff
@@ -805,3 +806,33 @@ class IPv46Field(StringField):
     def __init__(self, **options):
         options.setdefault('validators', []).append(validate_ipv46_address)
         super(IPv46Field, self).__init__(**options)
+
+
+class UUIDField(StringField):
+    """
+    An universally unique identifier.
+
+    Validates that the string represents a universally unique identifier.
+    """
+    data_type_name = "UUID"
+
+    def __init__(self, **options):
+        options.setdefault('validators', []).append(validate_uuid)
+        super(UUIDField, self).__init__(**options)
+
+    def to_python(self, value):
```

Comment:
This method needs to handle ValueError exceptions during decode, and type conversion.